### ISSUE TITLE
Inserting a Timezone aware `datetime.datetime` fails.

### ISSUE THREAD

**Issue by tonymillion** [chanzuckerberg/sqlalchemy-aurora-data-api#7]
It seems like the code is not converting the type to a string. I see the following error:

`Invalid type for parameter parameters[3].value.stringValue, value: 2019-12-21 16:59:27.415527+00:00, type: <class 'datetime.datetime'>, valid types: <class 'str'>`

I see the following SQL generated:

`'INSERT INTO verifications (id, verification_type, status, code_expires, number, sid) VALUES (CAST(:id AS UUID), :verification_type, :status, :code_expires, :number, :sid)'`

and the params look like:

``` 
{code_expires: datetime.datetime(2019, 12, 21, 16, 59, 27, 415527, tzinfo=datetime.timezone.utc), 
id: 'ca11e967-e90e-45df-9614-71316bb3581b', 
number: '<snip>', 
status: 0, 
sid: None, 
verification_type: 1}
```

Everything else was successfully converted to the correct type (including `id` which was a UUID).

**Comment by ludaavics** [comment]
Experiencing the same. Looks like it might be all date types (https://github.com/chanzuckerberg/aurora-data-api/issues/5)

**Comment by kislyuk** [comment]
This should be fixed in v0.1.5, please test.
